src/basic_bot/services/central_hub.py

```diff
@@ -251,14 +253,14 @@ async def handle_message(websocket):
         await websocket.close()
 
 
-async def persist_state_task():
+async def persist_state_task() -> None:
 
     while True:
         hub_state.persist_state()
         await asyncio.sleep(1)
 
 
-async def main():
+async def main() -> None:
     log.info("Loading persisted state")
     hub_state.init_persisted_state()
     log.info(f"Starting server on port {constants.BB_HUB_PORT}")
```

Comment:
Unrelated to this PR.  I think it's time to decompose central_hub into submodules a little if possible.